games/p.yaml

```diff
@@ -486,6 +499,364 @@
   - 'https://raw.githubusercontent.com/pokemium/PokeTraveler/master/docs/demo3.png'
   - 'https://raw.githubusercontent.com/pokemium/PokeTraveler/master/docs/demo4.png'
 
+- name: Pop3-Scripts
+  originals:
+  - 'Populous: The Beginning'
+  type: tool
+  repo: 'https://github.com/MrKosjaK/Pop3-Scripts'
+  development: halted
+  langs:
+  - Lua
+  licenses:
+  - Custom
+  info: No licence specified
+  updated: '2023-12-13'
+
+- name: pop3dobj
+  originals:
+  - 'Populous: The Beginning'
+  type: tool
+  repo: 'https://github.com/OpenPop/pop3dobj'
+  development: halted
+  content: commercial
+  langs:
+  - C++
+  licenses:
+  - Custom
+  info: 'Public domain; see https://github.com/OpenPop/notes#alacn'
+  updated: '2023-12-13'
+
+- name: popdisp
+  originals:
+  - 'Populous: The Beginning'
+  type: tool
+  repo: 'https://github.com/OpenPop/popdisp'
+  development: halted
+  content: swappable
+  langs:
+  - Component Pascal
+  licenses:
+  - Custom
+  info: 'Public domain; see https://github.com/OpenPop/notes#alacn'
```

Comment:
please add info on what this tool does
for a few other tools like "PopSoundEditor" it is obvious but for this one it is not